doc/user/layouts/partials/head.html

```diff
@@ -86,6 +86,20 @@
   });
 </script>
 
+<script
+  async
+  src="https://widget.kapa.ai/kapa-widget.bundle.js"
+  data-website-id="8afe5064-c908-44bd-8cd9-805d9e740fb9"
+  data-project-name="Materialize"
+  data-project-color="#472E85"
+  data-project-logo="https://avatars.githubusercontent.com/u/47674186?s=200&v=4"
+  data-modal-title="Materialize AI"
```

Comment:
It's sort of weird to call this "Materialize AI". Can we use "Ask AI" here as well?

Reply:
What feels weird about it? The kapa.ai default is to call it "XXX Docs AI"...

<img width="671" alt="image" src="https://github.com/MaterializeInc/materialize/assets/882976/1a9ea39d-7db1-4264-8ef6-266041de6d2f">

...which would be "Materialize Docs AI" for us. But I figured we'd want to integrate this in product at some point at which point it's nice to have a single name for it. The "XXX AI" scheme is also what Cockroach uses:

<img width="670" alt="image" src="https://github.com/MaterializeInc/materialize/assets/882976/29eccb3d-1475-4be3-96c4-429619298880">

Reply:
"Materialize AI" makes it sound like it's an AI product built on Materialize + I (personally) find it obnoxious to have the AI label all over the place. We could use something more neutral like "MZ Docs Assistant", which by extension could be "MZ Console Assistant" in the console, and wherever else we embed Kapa.

Reply:
I'm also an AI skeptic at heart but the ask that led us to kapa.ai was to lean more into AI instead of shying away from it. Calling the assistant "Foo AI" seems to be reasonably common, e.g., Pulumi AI:

<img width="1572" alt="image" src="https://github.com/MaterializeInc/materialize/assets/882976/122e44a8-a646-4036-b28b-d2f8eaa4f5a0">

However Elastic at least calls it the "[Elastic AI Assistant](https://www.elastic.co/elasticsearch/ai-assistant)":

<img width="967" alt="image" src="https://github.com/MaterializeInc/materialize/assets/882976/32f28c83-7468-4a79-a022-3938228a55f1">

I don't care terribly, and about to hand this project off to the product/console teams, but did want to flag that there is some forthcoming company strategy to embrace "AI" labels where appropriate.

Reply:
Not something I care terribly about, either, so I'll just disagree and commit with whatever lines up with the company strategy.